backend/src/main/java/com/bakdata/conquery/models/config/CsvResultRenderer.java

```diff
@@ -113,9 +112,6 @@ public Response createResult(Subject subject, ManagedExecution<?> execRaw, Datas
 			catch (Exception e) {
 				throw new WebApplicationException("Failed to load result", e);
 			}
-			finally {
-				onClose.run();
-			}
```

Comment:
In ResultArrowProcessor.java hast du noch eine finale log message drinne wenn alle Daten übertragen sind. Die ist ganz nice